TodoListPosta/app/containers/App.js

```diff
@@ -0,0 +1,87 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux'
+import {Platform, StyleSheet, Text, View} from 'react-native';
+
+import Title from '../components/title/index';
+import Footer from '../components/footer/index';
+import List from '../components/list/index';
```

Comment:
Importing `/index` is not necessary :smile: Is the same compared to:
```
import Title from '../components/Title';
```
Remember to put folder names in capital letter when they represent components !